storage/innobase/fsp/fsp0fsp.cc

```diff
@@ -1139,7 +1139,9 @@ fsp_enable_encryption(
 
```

Comment:
at the top ut_ad(mutex_own(&fil_system->mutex)); ?

Reply:
actually fil_sys is not acquired by the caller. There seems to be a bug below with extra "mutex_exit()" on failure

Reply:
removed that extra mutex_exit()